extensions/amp-access/0.1/login-dialog.js

```diff
@@ -267,7 +267,7 @@ export class WebLoginDialog {
     }
     dev().fine(TAG, 'Login done: ', result, opt_error);
     if (opt_error) {
-      this.reject_(opt_error);
+      this.reject_(user().createError('login failed', opt_error));
```

Comment:
Why? Isn't error already marked correctly as a user error or dev error? Why are we changing the meaning of the error?